projects/core/src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -940,6 +940,32 @@ function urlEncode(str)
     return str
 end
 
+--- Splits a string in a table.
+--
+-- @tparam string string_to_split The string to split.
+-- @tparam string pattern At what pattern to split the string.
+-- @treturn table A table containing the splitted strings.
+-- @usage args = textutils.splitString("arg1 arg2", " ")
+-- @since edit this if it goes live (I hope it does)
+local function split(string_to_split, pattern)
```

Comment:
I would recommend a much simpler split function.

[From StackOverflow](https://stackoverflow.com/questions/1426954/split-string-in-lua):

```lua
local function split(inputstr, sep)
        if sep == nil then
                sep = "%s"
        end
        local t={}
        for str in string.gmatch(inputstr, "([^"..sep.."]+)") do
                table.insert(t, str)
        end
        return t
end
```